package.json

```diff
@@ -54,16 +54,19 @@
     "check-disk-space": "^2.1.0",
     "cyclic-32": "^1.1.0",
     "debug": "^3.1.0",
+    "docker-parse-image": "^3.0.1",
```

Comment:
is this used anywhere? I see only the own implementation imported.

Reply:
maybe not, that comes from the work or Rose, I'll have to double check

Reply:
yep I forgot but I actually rewrote the important parts of that library in TS 😅